src/CentreonRemote/Tests/Infrastructure/Export/ExportParserJsonTest.php

```diff
@@ -31,15 +31,25 @@
  */
 class ExportParserJsonTest extends TestCase
 {
+    /**
+     *
+     * @var FileSystem
+     */
+    private $fs;
+
+    /**
+     *
+     * @var ExportParserJson
+     */
+    private $parser;
 
     public function setUp(): void
     {
         // mount VFS
         $this->fs = FileSystem::factory('vfs://');
         $this->fs->mount();
-        $this->fs->get('/')->add('tmp', new Directory([]));
-
-        $this->parser = new ExportParserJson;
+        $this->fs->get('/')->add('tmp', new Directory([])); /** @phpstan-ignore-line */
```

Comment:
To be discussed

Reply:
we can ignore this error cause we'll don't waste time to refactor this test